sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,24 +1,28 @@
 # Release History
 
-## 5.0.0-preview.8 (Unreleased)
+## 5.0.0 (2020-01-08)
 
+- This release marks the general availability of the `@azure/event-hubs` package.
 - Fixed potential issues with claims being mismanaged when subscriptions terminate.
 - Improved reporting of errors that occur when attempting to claim partitions from CheckpointStores.
 - Updated to use the latest version of the `@azure/core-amqp` package.
   This update allows the SDK to detect when a connection has gone idle for 60 seconds and attempt to reconnect.
 
-
 ### Breaking changes:
-  - Starting event positions are now passed in the `options` to the `subscribe()` method instead of using 
+
+- Starting event positions are now passed in the `options` to the `subscribe()` method instead of using
   the `processInitialize()` callback.
-  - The `MessagingError` class is updated to have the `code` property instead of `name` to contain the error
+- The default starting event position is now `latest` instead of `earliest`.
```

Comment:
`latest` and `earliest` are no longer functions and dont refer to any exported symbol from the package.

Reply:
How about this:

> If no position is passed and no checkpoints are available for the partition, the `subscribe()` method will start receiving events that are queued after the method invocation. This is different from the last preview, where events were received from the beginning of the partition.